### PR TITLE
Ot153 59 donations and thanks

### PR DESCRIPTION
**Pull request by fsalasevicius**
## JIRA
[OT153-118](https://alkemy-labs.atlassian.net/browse/OT153-118)

## Short description
Add styles in donations and fix errors with MP
Please add a short description about the changes.

## Screenshot or Video
donaciones

Please add a screenshot or video.

## Teammate collaborator

- [ ] Someone from the team collaborated (**_if so, please tag them after this line_**).

### Sanity Check Steps

- [x] All console logs have been removed.
- [x] This change was tested on my localhost.
- [ ] This change depends for another ticket.
- [x] All linked JIRA acceptance criteria are satisfied.
- [ ] Does this change require environment variables.

### CODE REVIEW

File: src/app/features/public/pages/donations/donations.component.html [hunk 1]
```diff
@@ -1,33 +1,38 @@
...
+      <div class="row justify-content-center">
+  
+        <div class="col col-12 col-sm-12 col-md-10 col-lg-12 mb-4 module-one_card option-selection-card active-card">
+          <div class="BoxNewDonation Bg-White w-75 p-3 p-lg-5 pb-5 m-auto">
+            <form [formGroup]="form"  (ngSubmit)="donation()">
+            <h4 class="Title BoxTitleDonation text-center Cl-GrisOscuro mb-4">Hacé tu donación ahora</h4>
+  
+            <div class="BoxTypeDonation BoxTypeDonation-Mensual mt-4 mt-lg-5 option-selection option-selection-module-mensual active">
+              <div>
+                <a (click)="donar1()"  class="BtnDonationImport BtnDefault">$500</a>
```
Comment (RicardoMaldonado93) [inline src/app/features/public/pages/donations/donations.component.html]: Arreglar la funcion repetida por cada input

File: src/app/features/public/pages/donations/donations.component.ts [hunk 2]
```diff
@@ -21,40 +25,98 @@ export class DonationsComponent implements OnInit {
...
-      return false;
-    } else {
-      this.importe = parseFloat(this.form.controls['value'].value);
+  hiddenInput(){
+    let others:any = document.getElementById('value');
+    others.type = 'hidden';
+    this.donationVariable = false;
+    }
+  
+    donar1(){
```
Comment (RicardoMaldonado93) [inline src/app/features/public/pages/donations/donations.component.ts]: refactorizar todas las funciones con donarX()

**Review by RicardoMaldonado93 — CHANGES_REQUESTED** [review]
Please fix the observations to continues with the PR

**Comment by fsalasevicius** [comment]
modificacion donaciones
Optimized functions

**Review by RicardoMaldonado93 — APPROVED** [review]
Amazing!

**Pull request merged**